### ISSUE TITLE
ChoiceGroup: no alt defined for options with images

### ISSUE THREAD

**Issue by prmittal** [OfficeDev/office-ui-fabric-react#3733]
#### Describe the issue:

No alt value is defined for choice group images.

![image](https://user-images.githubusercontent.com/11937034/35038122-c1cd7a44-fb9f-11e7-981b-4324ea8f76cc.png)

#### Actual behavior:

![image](https://user-images.githubusercontent.com/11937034/35038113-b65d9c2a-fb9f-11e7-9cc1-04c9576fefa0.png)

#### Expected behavior:

A valid alt value should be present. See https://dequeuniversity.com/rules/axe/2.6/image-alt for rule details.

**Comment by oengusmacinog** [comment]
This has been resolved by pull request https://github.com/OfficeDev/office-ui-fabric-react/pull/3742 in reguards to issue "#1026 ChoiceGroup: Support image alt text", so I am closing this issue.
